.github/workflows/ci.yaml

```diff
@@ -164,6 +164,20 @@ jobs:
           api-level: ${{ matrix.api-level }}
           script: ./gradlew connectedAndroidTest
 
+  android-delete-artifacts:
```

Comment:
I would add a comment explaining why we're doing this.

Reply:
Done